target/linux/ar71xx/image/atheros.mk

```diff
@@ -0,0 +1,11 @@
+define Profile/ANONABOX_PRO
```

Comment:
That's really not how files under target/linux/ar71xx/image work. Drop this file.